README.md

```diff
@@ -26,7 +26,10 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose (person)
+  person = "young prince"
+  puts "Would you take this rose and help out an old beggar, #{person}?"
+end
 ```
```

Comment:
this works, but how would it look if we passed in an argument when we call this method?